README.md

```diff
@@ -51,30 +51,27 @@ and processor architecture listed below.
 
 * CentOS 7
 * CentOS 8
-* Debian 9
 * Debian 10
 * FreeBSD 11.4
 * FreeBSD 12.2
-* Ubuntu 16.04
 * Ubuntu 18.04
 * Ubuntu 20.04
 
 ### Supported database engine versions
 
-Operating System | MariaDB         | PostgreSQL
----------------- | ----------------| -----------
-CentOS 7         | 5.5             | 9.3 
-CentOS 8         | 10.3            | 10.14 
-Debian 9         | 10.1            | 9.6
-Debian 10        | 10.3            | 11.7
-FreeBSD 11.4     | 5.7 (#)         | 12.4
-FreeBSD 12.2     | 5.7 (#)         | 12.4
-Ubuntu 16.04     | 10.0            | 9.5
-Ubuntu 18.04     | 10.1            | 10.14
-Ubuntu 20.04     | 10.3            | 12.4
+Operating System | MariaDB | PostgreSQL | SQLite
+---------------- | --------| -----------|------------------------
+CentOS 7         | 5.5     | 9.3        | 3.7.17 (?)
+CentOS 8         | 10.3    | 10.14      | 3.34 (?)
+Debian 10        | 10.3    | 11.7       | 3.27.2 (?)
+FreeBSD 11.4     | 5.7 (#) | 12.4       | 3.32.3 (##)
+FreeBSD 12.2     | 5.7 (#) | 12.4       | 3.32.3 (##)
+Ubuntu 18.04     | 10.1    | 10.14      | 3.22.0 (?)
+Ubuntu 20.04     | 10.3    | 12.4       | 3.33.0 (?)
 
-#) FreeBSD uses MySQL, not MariaDB.
+(#) FreeBSD uses MySQL, not MariaDB.
 
+(##) FreeBSD bundles SQLite in Perl DBD::SQLite (version 1.66).
```

Comment:
When is this footnote useful?

Reply:
If you install SQLite 3 as package on FreeBSD you will get version 3.34.0, but the bundled SQLite (3.32.3) is still used.